examples/v1beta1/kind-cluster/deploy.sh

```diff
@@ -53,6 +53,12 @@ kubectl get nodes
 echo -e "\nDeploying Katib components\n"
 kubectl apply -k "github.com/kubeflow/katib.git/manifests/v1beta1/installs/katib-standalone?ref=master"
 
+# If the local machine's CPU architecture is arm64, rewrite mysql image.
+if [ "$(uname -m)" = "arm64" ]; then
+  kubectl patch deployments -n kubeflow katib-mysql --type json -p \
+    '[{"op": "replace", "path": "/spec/template/spec/containers/0/image", "value": "arm64v8/mysql:8.0.29-oracle"}]'
```

Comment:
Is there any better replacement solution with `kustomize` itself?

Reply:
Since we have only [3 requirement tools](https://github.com/kubeflow/katib/tree/master/examples/v1beta1/kind-cluster#prerequisites) to work on this KinD cluster example, I added that code not to make increase requirement tools.